grudge/discretization.py

```diff
@@ -304,33 +313,32 @@ def connection_from_dds(self, from_dd, to_dd):
         else:
             raise ValueError("cannot interpolate from: " + str(from_dd))
 
-    def group_factory_for_quadrature_tag(self, quadrature_tag):
+    def group_factory_for_quadrature_tag(self, discretization_tag):
         """
         OK to override in user code to control mode/node choice.
         """
+        if discretization_tag is None:
+            discretization_tag = DISCR_TAG_BASE
 
-        if quadrature_tag is None:
-            quadrature_tag = QTAG_NONE
-
-        return self.quad_tag_to_group_factory[quadrature_tag]
+        return self.quad_tag_to_group_factory[discretization_tag]
 
     @memoize_method
-    def _quad_volume_discr(self, quadrature_tag):
+    def _quad_volume_discr(self, discretization_tag):
```

Comment:
Rename this method?

Reply:
Done: https://github.com/inducer/grudge/pull/83/commits/d4fedd7a1acba4dab4038dac349b36b71a7ca932